api/metrics/src/main/java/io/opentelemetry/api/metrics/LongUpDownCounter.java

```diff
@@ -5,62 +5,46 @@
 
 package io.opentelemetry.api.metrics;
 
-import io.opentelemetry.api.metrics.common.Labels;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.context.Context;
 import javax.annotation.concurrent.ThreadSafe;
 
-/**
- * UpDownCounter is a synchronous instrument and very similar to Counter except that Add(increment)
- * supports negative increments. This makes UpDownCounter not useful for computing a rate
- * aggregation. The default aggregation is `Sum`, only the sum is non-monotonic. It is generally
- * useful for capturing changes in an amount of resources used, or any quantity that rises and falls
- * during a request.
- *
- * <p>Example:
- *
- * <pre>{@code
- * class YourClass {
- *   private static final Meter meter = OpenTelemetry.getMeterProvider().get("my_library_name");
- *   private static final LongUpDownCounter upDownCounter =
- *       meter.
- *           .longUpDownCounterBuilder("active_tasks")
- *           .setDescription("Number of active tasks")
- *           .setUnit("1")
- *           .build();
- *
- *   // It is recommended that the API user keep a reference to a Bound Counter.
- *   private static final BoundLongUpDownCounter someWorkBound =
- *       upDownCounter.bind("work_name", "some_work");
- *
- *   void doSomeWork() {
- *      someWorkBound.add(1);
- *      // Your code here.
- *      someWorkBound.add(-1);
- *   }
- * }
- * }</pre>
- */
+/** An up-down-counter instrument that records {@code long} values. */
 @ThreadSafe
-public interface LongUpDownCounter extends SynchronousInstrument<BoundLongUpDownCounter> {
-
+public interface LongUpDownCounter extends UpDownCounter {
   /**
-   * Adds the given {@code increment} to the current value.
+   * Records a value.
```

Comment:
I went for consistency across instruments here.   There's an 'oddity' here where users can actually use aggregators to change what happens when recording values.   E.g. *by default* this would add an increment to current value, but you can use a `View` to convert this to a histogram or other behavior, which is why I went with `record a value` consistently across instruments.